src/main/scala/intellij/haskell/external/repl/ProjectStackReplProcess.scala

```diff
@@ -47,20 +48,24 @@ private[repl] class ProjectStackReplProcess(project: Project) extends StackReplP
   }
 
   private def load(psiFile: PsiFile, moduleName: Option[String]): Option[(StackReplOutput, Boolean)] = {
-    val filePath = getFilePath(psiFile)
-    val moduleNameAction = ApplicationManager.getApplication.runReadAction(new Computable[Option[String]] {
-      override def compute(): Option[String] = {
-        moduleName.orElse(HaskellPsiUtil.findModuleName(psiFile))
+    if (!HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
@rikvdkleij Should I check console file in each place where `HaskellProjectUtil.isLibraryFile()` is used? There are at lease 10 places where `HaskellProjectUtil.isLibraryFile()` stands 😅